### DISCUSSION TITLE
Random Disconnects

### DISCUSSION THREAD

**Discussion by KerrangR** [nefarius/BthPS3#28] [Q&A]
Checking the warnings on Windows Event Viewer shows the bth adapter stops responding, no error messages and refuses to connect unless I do a full restart.

No amount output rate control fixes the issue. These issues werent present when I previously used SCP Toolkit (Same controller and BT dongle.
![image_2021-09-23_223434](https://user-images.githubusercontent.com/11544024/134609596-bc1ded8b-ead6-49f2-a6b5-9c1d1d603cba.png)
)

**Comment by nefarius** [comment]
I recommend getting a different Bluetooth dongle. BthPS3 depends on the Microsoft Bluetooth stack to work properly with the adapter, I can't do anything about errors outside of the driver.
